src/entities/PersistentSignedOrderEntity.ts

```diff
@@ -92,7 +92,6 @@ export class PersistentSignedOrderEntity extends SignedOrderEntity {
             signature?: string;
             remainingFillableTakerAssetAmount?: string;
             orderState?: OrderEventEndState;
-            createdAt?: string;
```

Comment:
didn't need this in the constructor since `createdAt` is filled by postgres on `default`.